deploy/AllocatorDeploy.sol

```diff
@@ -0,0 +1,97 @@
+// SPDX-FileCopyrightText: © 2023 Dai Foundation <www.daifoundation.org>
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity ^0.8.16;
```

Comment:
We could argue these ones should also be >=0.8.0 as they are designed to be imported in some other repo.

Reply:
Pretty similar to the interfaces case.

Reply:
I don't think the deploy files should be imported, only the init and the instances files.
The distinction is becuase it's done like this in the d3m repo (though honestly I don't mind changing all to >=0.8.0, might be more clean)..

Reply:
Yeah you are right, I was more meaning about the init file but that was already correct, so nvm, will resolve this one.